rbac/rook-monitor-role.yaml

```diff
@@ -6,6 +6,27 @@ rules:
 - apiGroups:
   - monitoring.coreos.com
   resources:
-  - '*'
+  - cephclients
```

Comment:
I am wondering if these resources really belongs to the `monitoring.coreos.com`. If I am not mistaken they belong to the `ceph.rook.io`.  @umangachapagain Can you pls guide @parth-gr What resources should come under the `monitoring.coreos.com`

Reply:
We only care about `servicemonitors` and `prometheusrules` AFAIR for monitoring.
Link to CRD def: https://github.com/prometheus-operator/kube-prometheus/tree/main/manifests/setup

Reply:
Okay thanks